lib/lita-reviewme/github.rb

```diff
@@ -0,0 +1,70 @@
+require 'delegate'
+
+module Lita
+  module Reviewme
+    class Github
+      class UnknownOwner < StandardError; end
+      class CannotPostComment < StandardError; end
+
+      extend Forwardable
+
+      attr_reader :config, :repo, :pr_id
+
+      delegate [
+        :github_access_token,
+        :github_comment_template,
+        :github_comment,
+        :github_request_review
+      ] => :config
+
+      def initialize(config, repo, pr_id)
+        @config = config
+        @repo = repo
+        @pr_id = pr_id
+      end
+
+      def owner
+        pull_request.user.login
+      rescue Octokit::Error
+        raise UnknownOwner
+      end
+
+      def assign(reviewer)
+        request_review(reviewer) if github_request_review
+        add_comment(reviewer) if github_comment
+
+        raise CannotPostComment if !github_request_review && !github_comment
+      rescue Octokit::Error => e
+        raise CannotPostComment
+      end
+
+      private
+
+      def client
+        @client ||= Octokit::Client.new(access_token: github_access_token)
+      end
+
+      def pull_request
+        @pull_request ||= client.pull_request(repo, pr_id)
+      end
+
+      def request_review(reviewer)
+        options = { accept: 'application/vnd.github.black-cat-preview' }
```

Comment:
I believe that technically, according to the `octokit` library, this feature is still in ["preview"](https://github.com/octokit/octokit.rb/blob/master/lib/octokit/client/reviews.rb#L20). Although, technically speaking, it should still work but we would [see warnings in the logs](https://github.com/octokit/octokit.rb/blob/master/lib/octokit/preview.rb#L31-L37).